tests/TestHelpers/WebDavHelper.php

```diff
@@ -113,6 +113,49 @@ public static function getFileIdForPath(
 
 		return $matches[1];
 	}
+	
+	/**
+	 * returns body for propfind
+	 *
+	 * @param array|null $properties
+	 *
+	 * @return string
+	 * @throws Exception
+	 */
+	public static function getBodyForPropfind($properties) {
```

Comment:
```suggestion
	public static function getBodyForPropfind(?array $properties):  string {
```

IMO we can declare the types these days.